src/include/sof/audio/selector.h

```diff
@@ -45,6 +53,20 @@ struct micsel_data {
 	struct ipc4_base_module_cfg base_cfg;
 	struct ipc4_audio_format output_format;
 };
+
+enum ipc4_selector_config_id {
+	IPC4_SELECTOR_COEFFS_CONFIG_ID = 0,
+};
+
+/** \brief IPC4 configuration to setup selection/mixing. */
+struct ipc4_selector_coeffs_config {
+	/* Unused fields, needed to keep the structure aligned with common layout */
+	uint16_t rsvd0;
+	uint16_t rsvd1;
+
+	/* Mixing coefficients in Q10 fixed point format. */
```

Comment:
Pls use doxygen format consistently for entire structures.

Reply:
Done.